### PR TITLE
add CSS to exception page - fixes #13747

### LINKED ISSUE #13747: Improve CSS on server error page
### Steps to reproduce

1. Install and configure 7.0.4 and owncloud 8.0.0
2. Bring down the DB box
3. Try to access 7.0.4 - get valid error message
4. Try to access 8.0.0 - get Internal Server Error message

### Expected behaviour

The user should get the same message (can't reach DB) in both cases

### Actual behaviour

User gets Internal Server Message in 8.0.0

Screen shot from 7.0.4:

p270

Screen shot from 8.0.0:

p271

### Server Host info:

Ubuntu 14.04
PHP 5.5.9

### Server configuration

ownCloud Enterprise Edition 8.0 beta 2 (testing)

### PR DESCRIPTION
**Pull request by MorrisJobke**
cc @jancborchardt @LukasReschke @DeepDiver1975 @jnfrmarks 

fixes #13747

### CODE REVIEW

**Comment by MorrisJobke** [comment]
Now looks like this:

bildschirmfoto von 2015-01-29 18-30-41

Debug mode:
bildschirmfoto von 2015-01-29 18-31-07

**Comment by LukasReschke** [comment]
Interesting approach – didn't think of a THAT easy solution - mine was involving base.php :fearful: 

:+1:

**Comment by MorrisJobke** [comment]
@LukasReschke I just fired up the debugger and stepped through. Then I added it where it fits best :P

**Comment by LukasReschke** [comment]
Pfff – real men begin with taking a look at base ;-)

**Comment by MorrisJobke** [comment]
> Pfff – real men begin with taking a look at base ;-)

That's where I set my first breakpoint :P

File: lib/private/template.php [hunk 1]
```diff
@@ -215,6 +215,9 @@ public static function printErrorPage( $error_msg, $hint = '' ) {
 	 * @param Exception $exception
 	 */
 	public static function printExceptionErrorPage(Exception $exception) {
+		// hack to allow the styling of the exception page
+		\OCP\Util::addStyle('core', 'styles');
```
Comment (DeepDiver1975) [inline lib/private/template.php]: using style() ans script() in the template? might not apear that hacky then? :speak_no_evil:

Comment (MorrisJobke) [inline lib/private/template.php]: Doesn't work, because this is just included in another code path and not loaded in the OC_Template class :(

Comment (MorrisJobke) [inline lib/private/template.php]: Weird. It should be loaded, but I tested and it didn't work and just throw more errors :(

Comment (MorrisJobke) [inline lib/private/template.php]: Let me check that again.

Comment (MorrisJobke) [inline lib/private/template.php]: Works now -.- I force pushed the proper version. I don't know what I've done wrong earlier.

**Comment by MorrisJobke** [comment]
@LukasReschke even better :)

File: lib/private/template.php (hunk 1 above)
Comment (DeepDiver1975) [inline lib/private/template.php]: > I don't know what I've done wrong earlier.

sounds like: call it a day - have a beer and good night :speak_no_evil:

**Comment by Xenopathic** [comment]
:+1:

**Pull request merged**